peru/main.py

```diff
@@ -221,6 +221,31 @@ def do_override(params):
             print('{}: {}'.format(module, params.runtime.get_override(module)))
 
 
+@peru_command('module', '''\
+Usage:
+    peru module <list> [-z]
+
+Options:
+    -z              print one module per line for machine-readable formatting
+
+''')
+def do_list(params):
+    modules = params.scope.modules.values()
+    output = ''
+    delim = ' | ' if params.args['-z'] else "\n  "
+
+    for module in modules:
+        output += module.name
+        for field, val in module.plugin_fields.items():
+            output += '{}{}: {}'.format(delim, field, val)
+        if(params.runtime.get_override(module.name)):
+            output += '{}{}: {}'.format(delim, 'override', params.runtime.get_override(module.name))
```

Comment:
It's currently allowed for a [user-defined module type](https://github.com/buildinspace/peru#creating-new-module-types) to have a field called `override`. (Maybe we should have some reserved words though?) It probably makes sense to distinguish different datatypes like module fields and override names more explicitly in the output, instead of just relying on the name.